dockstore-integration-testing/src/test/java/io/dockstore/webservice/NotebookIT.java

```diff
@@ -59,6 +67,9 @@ class NotebookIT extends BaseIT {
     @SystemStub
     public final SystemErr systemErrRule = new SystemErr(new NoopStream());
 
+    private final String installationId = "1179416";
```

Comment:
This is the same id as `AppToolHelper.INSTALLATION_ID` which is public. Use the same variable, may help us someday if we need to optimize API quotas for GitHub app API calls.